into/tests/test_pydot.py

```diff
@@ -6,4 +7,9 @@ def test_create_pydot_image():
         dot_graph()
     except:
         pass
-
+    finally:
+        for ext in ('pdf', 'png'):
+            try:
+                os.remove('conversions.%s' % ext)
+            except:
+                pass
```

Comment:
FWIW I actually intended this side effect.  At the moment often use the graph for debugging.  I admit that this is a bit dirty/invasive though.